types/read_config.go

```diff
@@ -92,9 +108,15 @@ func (ReadConfig) Read(hasEnv HasEnv) BootstrapConfig {
 
 // BootstrapConfig for the process.
 type BootstrapConfig struct {
-	EnableFunctionReadinessProbe bool
-	ReadTimeout                  time.Duration
-	WriteTimeout                 time.Duration
-	ImagePullPolicy              string
-	Port                         int
+	HTTPProbe                         bool
```

Comment:
Think this might be useful to comment on:

```
// HTTPProbe when set to true switches readiness and liveness probe to access /_/health over HTTP instead of accessing /tmp/.lock.
```